ocd_backend/transformers/greenvalley.py

```diff
@@ -27,57 +27,56 @@ def get_meeting_dates(self, meeting):
         start_date = None
         end_date = None
 
-        if meeting.get(u'bis_vergaderdatum', u'').strip() != u'':
+        if meeting.get('bis_vergaderdatum', '').strip() != '':
             start_date = datetime.fromtimestamp(
-                float(meeting[u'bis_vergaderdatum']) +
-                (float(meeting.get(u'bis_starttijduren', '0') or '0') * 3600) +
-                (float(meeting.get(u'bis_starttijdminuten', '0') or '0') * 60))
+                float(meeting['bis_vergaderdatum']) +
+                (float(meeting.get('bis_starttijduren', '0') or '0') * 3600) +
+                (float(meeting.get('bis_starttijdminuten', '0') or '0') * 60))
             end_date = datetime.fromtimestamp(
-                float(meeting[u'bis_vergaderdatum']) +
-                (float(meeting.get(u'bis_eindtijduren', '0') or '0') * 3600) +
-                (float(meeting.get(u'bis_eindtijdminuten', '0') or '0') * 60))
-        elif u'publishdate' in meeting:
+                float(meeting['bis_vergaderdatum']) +
+                (float(meeting.get('bis_eindtijduren', '0') or '0') * 3600) +
+                (float(meeting.get('bis_eindtijdminuten', '0') or '0') * 60))
+        elif 'publishdate' in meeting:
             start_date = datetime.fromtimestamp(
-                float(meeting[u'publishdate']))
+                float(meeting['publishdate']))
             end_date = datetime.fromtimestamp(
-                float(meeting[u'publishdate']))
+                float(meeting['publishdate']))
 
         return start_date, end_date
 
     def _get_documents_as_media_urls(self, original_item):
         media_urls = {}
-        if u'attachmentlist' in original_item:
-            for att_key, att in original_item.get(u'attachmentlist', {}).iteritems():
-                if att[u'objecttype'] == 'AGENDAPAGE':
+        if 'attachmentlist' in original_item:
+            for att_key, att in original_item.get('attachmentlist', {}).items():
+                if att['objecttype'] == 'AGENDAPAGE':
                     continue
 
-                url = "https://staten.zuid-holland.nl/dsresource?objectid=%s" % (
-                    att[u'objectid'].encode('utf8'),)
+                url = "https://staten.zuid-holland.nl/dsresource?objectid=%s" % att['objectid']
 
-                doc_hash = unicode(
-                    sha1(url + ':' + att[u'objectname'].encode('utf8')).hexdigest())
+                doc_hash = str(
+                    sha1(url.encode('utf-8') + ':'.encode('utf-8') + att['objectname'].encode('utf-8')).hexdigest()
```

Comment:
Perhaps this can be refactored?

Reply:
I can reformat it, but all strings have to be `utf-8` encoded for `hexdigest()` to work.

Reply:
Sounds good.